inc/i2c_frame.h

```diff
@@ -1,7 +1,6 @@
 /****************************************************************************
  *
- *   Copyright (C) 2013 Fortiss An-Institut TU Munchen All rights reserved.
- *   Author: Thomas Boehm <thomas.boehm@fortiss.org>
```

Comment:
If you remove author attribution in the header, please add it back as doxygen comment